INTERPRETING.md

```diff
@@ -34,6 +34,20 @@ properties of the global scope prior to test execution.
 - **`print`** A function that exposes the string value of its first argument to
   the test runner. This is used as a communication mechanism for asynchronous
   tests (via the `async` flag, described below).
+- **`$`** An ordinary object with the following properties:
+  - **`createRealm`** - a function which creates a new [ECMAScript
+    Realm](http://www.ecma-international.org/ecma-262/6.0/#sec-code-realms),
+    defines this API on the new realm's global `this` value, and returns the
+    `$` property of the new realm's global `this` value
+  - **`detachArrayBuffer`** - a function which implements [the
+    DetachArrayBuffer abstract
+    operation](http://www.ecma-international.org/ecma-262/6.0/#sec-detacharraybuffer)
+  - **`evalScript`** - a function which accepts a string value as its first
```

Comment:
Well, you have to pick which semantics you want. Consider the code running in realm B that does `realmA$.global` or `realmA$.evalScript`. Do you want that to get the global of realm A, or realm B? Do you want it to eval script in realm A, or in realm B?